src/main/java/com/smartsheet/api/models/ReportRow.java

```diff
@@ -0,0 +1,52 @@
+package com.smartsheet.api.models;
+
+/*
+ * #[license]
+ * Smartsheet SDK for Java
+ * %%
+ * Copyright (C) 2014 Smartsheet
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * %[license]
+ */
+
+/**
+ * Represents the ReportRow object.
+ */
+public class ReportRow extends AbstractRow<ReportColumn, ReportCell>{
```

Comment:
We do not need to add the sheetId & getter/setter as these already exist in the AbstractRow. Please relay this info to Eric as well since the C# SDK is doing the same thing.